lib/api/controllers/documentController.js

```diff
@@ -360,48 +389,48 @@ function DocumentController (kuzzle) {
 
         return deletedIds;
       });
-  };
+  }
 
   /**
    * Delete several documents matching a query through the persistent layer
    *
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.deleteByQuery = function documentDeleteByQuery (request) {
+  deleteByQuery(request) {
     assertHasBody(request);
     assertBodyHasAttribute(request, 'query');
     assertBodyAttributeType(request, 'query', 'object');
     assertHasIndexAndCollection(request);
 
-    return engine.deleteByQuery(request)
+    return this.engine.deleteByQuery(request)
       .then(response => {
-        kuzzle.notifier.notifyDocumentDelete(request, response.ids);
+        this.kuzzle.notifier.notifyDocumentDelete(request, response.ids);
 
         return response;
       });
-  };
+  }
 
   /**
    * Validate a document against collection validation specifications.
    *
    * @param {Request} request
    * @returns {Promise<Object>}
    */
-  this.validate = function documentValidate (request) {
+  validate(request) {
     assertHasBody(request);
     assertHasIndexAndCollection(request);
     assertHasId(request);
 
-    return kuzzle.validation.validationPromise(request, true)
+    return this.kuzzle.validation.validationPromise(request, true)
       .then(response => {
         if (!response.valid) {
-          kuzzle.pluginsManager.trigger('validation:error', `The document does not comply with the ${request.input.resource.index} / ${request.input.resource.collection} : ${JSON.stringify(request.input.body)}`);
+          this.kuzzle.pluginsManager.trigger('validation:error', `The document does not comply with the ${request.input.resource.index} / ${request.input.resource.collection} : ${JSON.stringify(request.input.body)}`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 217 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)